packages/fsapp/src/beta-app/modal/types.ts

```diff
@@ -13,6 +13,7 @@ export interface ModalService {
 export interface ModalOptions extends RNModalOptions, ModalProps {
   readonly backdropStyle?: StyleProp<ViewStyle>;
   readonly title?: string;
+  navigationOptions?: Options;
```

Comment:
This should probably be `readonly` like the other options.
Not a big deal, just a preference to assure values are not unexpectedly changed.

Reply:
It feels a bit strange to be able to set the `RNModalOptions` both directly on `ModalOptions` and in `navigationOptions`.
Granted removing it from `ModalOptions` would be a breaking change, but this will probably be refactored before fs12 to clean up the interface.